charts/gardener-extension-shoot-rsyslog-relp/templates/clusterrole.yaml

```diff
@@ -58,29 +58,3 @@ rules:
   - create
   - list
   - watch
-- apiGroups:
```

Comment:
You could reflect the split in `example/25-rbac.yaml`. Alternatively, you can only leave controller-registration.yaml and shoot.yaml under `example/`. I don't think the other examples will be used because now we have local setup that is based on gardener's local setup. Hence, there is no need anymore to create CRDs, Cluster, RBAC manually anymore.

Reply:
Yep, I planned to clean up the examples in a separate PR that's why i gave up with updating the rbac rules there.
Let me add that as a small task in #2

Reply:
Okay, if you prefer to do it via separate PR.
IMO you could also amend to this PR as you anyways drop the old local setup based rules from the Makefile.
Anyways, I am approving. The final decision in yours.